recipe_scrapers/rewe.py

```diff
@@ -0,0 +1,76 @@
+import ast
+import re
+
+from ._abstract import AbstractScraper
+
+
+class Rewe(AbstractScraper):
+    @classmethod
+    def host(cls):
+        return "rewe.de"
+
+    def language(self):
+        return self.soup.find("html")["lang"]
+
+    def author(self):
+        return ""
```

Comment:
When the website credits authors for their recipes, I think we should include that information here too.

I'd recommend either removing this method -- allowing our `SchemaOrgFill` plugin to populate the data, or calling `self.schema.author()` to retrieve that in the code here.